### ISSUE TITLE
PLCrashReporter does not report on crashes if Crashlytics is running

### ISSUE THREAD

**Issue by CanfieldMobileDev** [microsoft/plcrashreporter#224]
### **Description**
When I saw this bullet point under Features: "_Easy to integrate with existing or custom crash reporting services._", I assumed that I could still use PLCrashReporter even if my app integrates Firebase/Crashlytics framework. But it appears that it is not the case and PLCrashReporter does not detect crashes. Could you please elaborate on whether or not this is by design and, if so, what was meant by that feature description.

Thank you

### **Repro Steps**

N/A

### **Details**

1. Which SDK version are you using? - v1.8.1
2. Which OS version did you experience the issue on? - iOS 15
3. Which CocoaPods/Carthage/Xcode version are you using?
4. What device version did you see this error on? Were you using an emulator or a physical device? - iPad
5. What language are you using? - both
    - [ ] Objective C
    - [ ] Swift
6. What third party libraries are you using?

**Comment by AnastasiaKubova** [comment]
Hi! Thanks for getting in touch with us!
`PLCrashReporter` doesn't work correctly with other Crash frameworks in parallel. We don't provide support for this scenario.
Duplicate of https://github.com/microsoft/plcrashreporter/issues/158.

**Comment by CanfieldMobileDev** [comment]
Thanks for the info. However, given your answer, I still must ask you. What does this sentence listed under Features mean: "_Easy to integrate with existing or custom crash reporting services_."?

Also, since I posted my original question last week, I've made some progress with my testing. What I am observing when I crash test my test app is that I receive crash reports both from Crashlytics installed at the app level and PLCrashReporter installed in a framework. Perhaps you could comment on why that happens, if it's not supposed to. Thank you.

**Comment by AnastasiaKubova** [comment]
This point is outdated. Thank you for noticing this, we will remove the point from Readme.
You can try to use it in the project with other crash reporters, but I'm not sure that it will work correctly.

**Comment by CanfieldMobileDev** [comment]
Understood. That line did sound promising. Too bad we didn't know at the time that it was outdated :\. Thanks for your help though.
